tests/e2e/e2e_test.go

```diff
@@ -52,6 +51,99 @@ func (s *IntegrationTestSuite) TestIBCTokenTransfer() {
 		s.Require().NotEmpty(ibcStakeDenom)
 	})
 
+	s.Run("send_umee_to_gaia", func() {
+		// require the recipient account receives the IBC tokens (IBC packets ACKd)
+		var (
+			supply sdk.Coins
+			err    error
+		)
+
+		token := sdk.NewInt64Coin(appparams.BondDenom, 100000000) // 100UMEE
+		// send 100UMEE to umee to gaia
+		// Note: receiver is null means hermes will default send to key_name (from config)
+		s.sendIBC(s.chain.id, gaiaChainID, "", token)
+
+		gaiaAPIEndpoint := fmt.Sprintf("http://%s", s.gaiaResource.GetHostPort("1317/tcp"))
+		umeeAPIEndpoint := fmt.Sprintf("http://%s", s.valResources[0].GetHostPort("1317/tcp"))
+		s.Require().Eventually(
+			func() bool {
+				supply, err = queryTotalSupply(gaiaAPIEndpoint)
+				s.Require().NoError(err)
+				return supply.Len() == 2
+			},
+			time.Minute,
+			5*time.Second,
+		)
+
+		ibcDenom := "ibc/9F53D255F5320A4BE124FF20C29D46406E126CE8A09B00CA8D3CFF7905119728"
+		for _, c := range supply {
+			// 9F53D255F5320A4BE124FF20C29D46406E126CE8A09B00CA8D3CFF7905119728 = denom-hash(transfer/channe-0/uumee)
+			if c.Denom == ibcDenom {
+				s.Require().Equal(token.Amount.Int64(), c.Amount.Int64())
+				break
+			}
+		}
+
+		// sending back some amount from receiver to sender (ibc/XXX)
+		s.sendIBC(gaiaChainID, s.chain.id, "", sdk.NewInt64Coin(ibcDenom, 1000))
+		// check the ibc (umee) quota after ibc txs
+		s.Require().Eventually(
+			func() bool {
+				outflows, err := queryOutflows(umeeAPIEndpoint, appparams.BondDenom)
+				s.Require().NoError(err)
+				outflow := outflows.AmountOf(appparams.BondDenom)
+				s.T().Logf("quota outflow of %s is %s ", appparams.BondDenom, outflow.String())
+				s.Require().True(outflow.GT(sdk.NewDec(0)))
+				return outflows.Len() == 1
+			},
+			time.Minute,
+			5*time.Second,
+		)
+		s.Require().Eventually(
+			func() bool {
+				supply, err = queryTotalSupply(gaiaAPIEndpoint)
+				s.Require().NoError(err)
+				return supply.Len() == 2
```

Comment:
we should check the value as well. Maybe we can query the value before the test, and after the tests and see if it's correct.